www/_posts/2023-07-10-cordova-ios-7.0.0.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova iOS 7.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 7.0.0`! This is one of Cordova's supported platforms for building iOS applications.
+
+* [cordova-ios@7.0.0](https://www.npmjs.com/package/cordova-ios)
+
+**Upgrade steps:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@7.0.0
+```
+
+**New install steps:**
+
+```bash
+cordova platform add ios@7.0.0
+```
+
+## Release Highlights
+
+### Breaking Changes
+
+* **Removal of `podspec` type from `framework` tag**
+
+    Since Cordova-iOS 5.0.0, the new `podspec` tag was added to improve the readability and support of features that CocoaPods provided.
+
+    This release officially removes the old `type="podspec"` implementation of the `framework` tag. If you maintain plugins and still using the `framework` tag to load pod specs, it is recommended to migrate to the newer implementation.
+
+    For implementation specifications, please see our [Apache Cordova - Podspec](https://cordova.apache.org/docs/en/dev/plugin_ref/spec.html#podspec-) docs.
+
+* **Removed default `CONFIGURATION_BUILD_DIR` overrides**
+
+    This will change the location of where the output files are generated.
+
+    * `build/emulator` will become `build/Debug-iphonesimulator`
+    * `build/device` will become `build/Release-iphoneos`
+
+    This will help ensure that debug and release files are never mixed up in the same directory and improves support for macOS `maccatalyst` builds.
+
+* **Node Support**
+
+    We have dropped support for Node 14.x and increase the minimum Node requirement to greater than or equal to 16.13.0.
```

Comment:
Slight preference to say NodeJS instead of just Node:

```suggestion
* **Update supported NodeJS versions**

    We have dropped support for NodeJS 14.x and increase the minimum NodeJS requirement to greater than or equal to 16.13.0.
```